CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+This project is [Apache 2.0 Licenced](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/LICENCE.txt) and welcomes external contributions. When contributing to this repository, please first discuss the change you wish to make via an [issue](https://github.com/ocp-power-automation/ocp4-upi-powervm/issues). You could also discuss via email, or use any other method with the owners of this repository before making a change.
```

Comment:
We can remove this statement "You could also discuss via email, or use any other method with the owners of this repository before making a change" for now. Once we add a slack channel we can update it.